README.md

```diff
@@ -1,8 +1,31 @@
 # Redux-cycles
+
+<div align="center">
+  <img src="logo.png" alt="Cycle.js + Redux = <3" />
```

Comment:
I would go with the first (redux on the left) to favour meaning over aesthetics. Yes, Good point on formatting and having a picture instead!